ext/intl/dateformat/dateformat_format_object.cpp

```diff
@@ -37,7 +37,7 @@ using icu::GregorianCalendar;
 using icu::StringPiece;
 using icu::SimpleDateFormat;
 
-static const DateFormat::EStyle valid_styles[] = {
+static constexpr DateFormat::EStyle valid_styles[] = {
```

Comment:
`constexpr` is a C++11 feature; I'm not sure whether we can require it.

Reply:
Isn't the issue that REHL 7 is still not EOL and the GCC version which comes bundled doesn't support C11/C++11 fully?

Reply:
Huh, well then I suppose it should be fine?